### PR TITLE
Added sensor ranges and units to RDM ModelCollector

### PR DESCRIPTION
**Pull request by vanous**
Example of collected data with these ranges in: [robin_dl7s_profile@robe.txt](https://github.com/OpenLightingProject/ola/files/1107996/robin_dl7s_profile.robe.txt)

thank you

Petr

### CODE REVIEW

File: tools/rdm/ModelCollector.py [hunk 1]
```diff
@@ -239,6 +239,12 @@ def _HandleSensorData(self, data):
       'description': data['name'],
       'type': data['type'],
       'supports_recording': data['supports_recording'],
+      'range_min':data['range_min'],
+      'range_max':data['range_max'],
+      'normal_min':data['normal_min'],
+      'normal_max':data['normal_max'],
+      'unit':data['unit'],
+
```
Comment (peternewman) [inline tools/rdm/ModelCollector.py]: Can you drop the blank line please.

File: tools/rdm/ModelCollector.py [hunk 2]
```diff
@@ -239,6 +239,12 @@ def _HandleSensorData(self, data):
       'description': data['name'],
       'type': data['type'],
       'supports_recording': data['supports_recording'],
+      'range_min':data['range_min'],
```
Comment (peternewman) [inline tools/rdm/ModelCollector.py]: Can we have a space after the colon and before data please.

**Comment by vanous** [comment]
Sorry about format. Fixed.

**Comment by peternewman** [comment]
Retrospective review if you fancy it @nomis52 .

**Comment by peternewman** [comment]
On a side note @vanous I see you've got a lot more manufacturer specific PIDs listed in that file than we currently have:
http://rdm.openlighting.org/pid/manufacturer?manufacturer=21075

You should really add these to the system too, so people can use them to control the fixtures:
https://github.com/OpenLightingProject/rdm-app/blob/master/data/pid_data.py#L3238

**Pull request merged**